iib/web/models.py

```diff
@@ -678,6 +678,14 @@ def _from_json(
             request_kwargs, required_params=required_params, optional_params=optional_params,
         )
 
+        # Check if no bundles `from_index and `binary_image` are specified
```

Comment:
Could you clarify why `from_index` is needed? If omitted, wouldn't a new empty database be created?